datadog/tests/cassettes/TestAccDatadogSyntheticsTestMultistepApi_Basic.yaml

```diff
@@ -102,26 +102,26 @@ interactions:
},"retry":{"count":5,"interval":1000},"subtype":"http"},{"allowFailure":false,"assertions":[{"operator":"is","target":200,"type":"statusCode"}],"extractedValues":[],"isCritical":false,"name":"Second api step","request":{"allow_insecure":true,"basicAuth":{"accessTokenUrl":"https://token.datadoghq.com","audience":"audience","clientId":"client-id","clientSecret":"client-secret","scope":"scope","tokenApiAuthentication":"header","type":"oauth-client"},"follow_redirects":true,"httpVersion":"any","method":"GET","noSavingResponseBody":false,"persistCookies":false,"timeout":30,"url":"https://docs.datadoghq.com"},"subtype":"http"},{"allowFailure":false,"assertions":[{"operator":"is","target":200,"type":"statusCode"}],"extractedValues":[],"isCritical":false,"name":"Third api step","request":{"allow_insecure":true,"basicAuth":{"accessTokenUrl":"https://token.datadoghq.com","audience":"audience","clientId":"client-id","clientSecret":"client-secret","password":"password","resource":"resource","scope":"scope","tokenApiAuthentication":"body","type":"oauth-rop","username":"username"},"follow_redirects":true,"httpVersion":"any","method":"GET","noSavingResponseBody":false,"persistCookies":false,"timeout":30,"url":"https://docs.datadoghq.com"},"subtype":"http"},{"allowFailure":false,"assertions":[{"operator":"is","target":200,"type":"statusCode"}],"extractedValues":[],"isCritical":false,"name":"Fourth api step","request":{"allow_insecure":true,"basicAuth":{"password":"password","type":"digest","username":"username"},"follow_redirects":true,"httpVersion":"any","method":"GET","noSavingResponseBody":false,"persistCookies":false,"timeout":30,"url":"https://docs.datadoghq.com"},"subtype":"http"},{"allowFailure":false,"assertions":[{"operator":"is","target":1,"type":"grpcHealthcheckStatus"}],"extractedValues":[],"isCritical":false,"name":"gRPC health check step","request":{"allow_insecure":false,"callType":"healthcheck","host":"https://docs.datadoghq.com","message":"","metadata":{"foo":"bar"},"method":"","port":443,"service":"greeter.Greeter","timeout":0},"subtype":"grpc"},{"allowFailure":false,"assertions":[{"operator":"validatesJSONPath","target":{"elementsOperator":"firstElementMatches","jsonPath":"$.message","operator":"is","targetValue":"Hello, John!"},"type":"grpcProto"}],"extractedValues":[],"isCritical":false,"name":"gRPC behavior check step","request":{"allow_insecure":false,"callType":"unary","compressedProtoFile":"eJwqzs9NVSgoyi/JV0jLzEkFBAAA//8u0wXJ","host":"https://docs.datadoghq.com","message":"{\"name\": \"John\"}","metadata":{"foo":"bar"},"method":"SayHello","port":443,"service":"greeter.Greeter","timeout":0},"subtype":"grpc"},{"name":"Wait step","subtype":"wait","value":5}]},"locations":["aws:eu-central-1"],"message":"Notify @datadog.user","name":"tf-TestAccDatadogSyntheticsTestMultistepApi_Basic-local-1723819609","options":{"httpVersion":"any","min_location_failed":1,"tick_every":900},"status":"paused","subtype":"multi","tags":["multistep"],"type":"api"}
```

Comment:
In the request body generated by Terraform, notice that `VAR_EXTRACT_BODY` doesn't have a `field` property:

```jsonc
{
  ...
  "extractedValues": [
    {
      "name": "VAR_EXTRACT_BODY",
      "parser": {"type": "json_path", "value": "$.id"},
      "secure": false,
      "type": "http_body"
    },
    {
      "field": "content-length",
      "name": "VAR_EXTRACT_HEADER",
      "parser": {"type": "regex", "value": ".*"},
      "secure": true,
      "type": "http_header"
    }
  ]
}
```